src/Magento/FunctionalTestingFramework/Util/Script/ScriptUtil.php

```diff
@@ -98,15 +102,34 @@ public function getModuleXmlFilesByScope($modulePaths, $scope)
         return $found ? $finder->files() : [];
     }
 
+    /**
+     * Return suite XML files in TESTS_BP/ROOT_SUITE_DIR directory
+     *
+     * @return Finder|array
+     * @throws TestFrameworkException
+     */
+    public function getRootSuiteXmlFiles()
+    {
+        $rootSuitePath = FilePathFormatter::format(TESTS_BP) . self::ROOT_SUITE_DIR;
+        if (!realpath($rootSuitePath)) {
+            return [];
+        }
+        $finder = new Finder();
+        $finder->files()->followLinks()->in($rootSuitePath)->name("*.xml");
```

Comment:
added a suite with a deprecated test AdminLoginTest like this, should this be called out in the static-check?
IMO would helpful to callout suites which have direct references to deprecated tests i.e using  <test name=:..>
```
    <suite name="SampleSuite">
        <before>
            <comment userInput = "test" stepKey="comment"/>
        </before>
        <include>
            <test name="AdminLoginTest"/>
        </include>
        <after>
            <comment userInput = "test" stepKey="comment"/>
        </after>
    </suite>
```

Reply:
Good catch :)
Fixed

Reply:
Was expecting this Suite to show up in the report, the suite location is dev/tests/acceptance/tests/_suite shouldn't it be because of deprecated `AdminLoginTest`? Suites in app/code show up though.

Reply:
I merged in RC2 branch, please re-run the test.

Reply:
Hmm weird, I did re-run still no log for suite in dev/tests

Reply:
Nm. Now it should get fixed.